doc/user_guide/model.rst

```diff
@@ -241,6 +237,24 @@ the following template to suit your needs:
             """
             return x
 
+Define components from a fixed-pattern
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :py:class:`~._components.scalable_fixed_pattern.ScalableFixedPattern`
+component enables fitting a pattern (in the form of a
+:py:class:`~._signals.signal1d.Signal1D` instance) to data by shifting 
+(:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.shift`)
+and
+scaling it in the x and y directions using the
+:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.xscale`
+and
+:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.yscale`
+parameters respectively.
+
+``xscale`` and ``yscale`` parameters. The position of the pattern can also be adjusted
```

Comment:
There is a repetition here too!
https://hyperspy--2455.org.readthedocs.build/en/2455/user_guide/model.html#define-components-from-a-fixed-pattern

Reply:
Done, thanks!